src/sentry/services/http.py

```diff
@@ -87,6 +87,7 @@ def __init__(self, host=None, port=None, debug=False, workers=None,
         options['enable-threads'] = True
         options['lazy-apps'] = True
         options['single-interpreter'] = True
+        options['cache2'] = 'name=options,items=100,purge_lru=true'
```

Comment:
This is 100 total cache items, which allocates 6MB of space in uwsgi master process. We only need ~25, but I didn't want to pick a number that was too close to how many keys we needed that we'd constantly have to bump it.